estate_account/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models, Command
+
+
+class EstateProperty(models.Model):
+
+    _inherit = "estate.property"
+
+    def action_sold(self):
+        res = super().action_sold()
```

Comment:
```suggestion
        res = super().action_sold()
```
why to call super before and not at the end?

Reply:
We must call the parent method (super().action_sold()) first, as we cannot generate an invoice without a sold property and its corresponding offer. If we call it last, we won't have the selling price of the current property.